steam/webapi.py

```diff
@@ -213,9 +213,10 @@ def __call__(self, **kwargs):
         params = {}
         # process special case kwargs
         for param in ('key', 'format', 'raw'):
-            if param in kwargs:
-                params[param] = kwargs[param]
-                del kwargs[param]
+            if param not in kwargs:
+                parent_param = getattr(self._parent._parent, param, None)
```

Comment:
We don't need to do this, since it's done in `WebAPI._api_request()`

https://github.com/ValvePython/steam/blob/0daf05c0be5dd952d7975ead53d2d42df1e62fde/steam/webapi.py#L90-L92